src/components/navBar/navBar.module.scss

```diff
@@ -79,4 +78,107 @@ $theme-active: #87d870;
         color: $white;
         font-weight: 600;
     }
+
+    .hamburgerIcon {
+        display: none;
+    }
+    ul li {
+        list-style: none;
+    }
+    .navLinks ul {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        gap: 1rem;
+    }
+    .active {
+        color: $theme-active;
+        text-decoration: underline;
+        text-decoration-thickness: $thickness;
+        text-underline-offset: $offset;
+    }
+}
+@media (max-width: 970px) {
+    .navBar {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        // border: 2px solid red;
+        position: relative;
+        .navLinks {
+            a {
+                padding: 10px 40px;
+                text-align: left;
+                color: #1d1283;
+                display: none;
+            }
+        }
+    }
+
+    .signInLink {
+        position: absolute;
+        top: 10px;
+        right: 20px;
+    }
+
+    .showLogo {
+        display: none !important;
+    }
+
+    .hamburgerIcon {
+        display: inline-block !important;
+        height: 73px;
+        width: 73px;
+        padding: 18px;
+        cursor: pointer;
+        font-size: 20px;
+        color: white;
```

Comment:
use responsive units like rem and variable for colors

Reply:
ok will fix it